var/spack/repos/builtin/packages/py-requests/package.py

```diff
@@ -23,13 +23,19 @@ class PyRequests(PythonPackage):
         'requests.packages.urllib3.contrib._securetransport'
     ]
 
+    version('2.21.0', '1bcd0e0977c3f8db1848ba0e2b7ab904')
     version('2.14.2', '4c3c169ed67466088a2a6947784fe444')
     version('2.13.0', '921ec6b48f2ddafc8bb6160957baf444')
     version('2.11.1', 'ad5f9c47b5c5dfdb28363ad7546b0763')
     version('2.3.0',  '7449ffdc8ec9ac37bbcd286003c80f00')
 
     depends_on('py-setuptools', type='build')
 
+    depends_on('py-certifi@2017.4.17:', type='run')
```

Comment:
This version does not seem to exist?
```
❯ spack info py-certifi
PythonPackage:   py-certifi

Description:
    Certifi: A carefully curated collection of Root Certificates for
    validating the trustworthiness of SSL certificates while verifying the
    identity of TLS hosts.

Homepage: http://certifi.io/

Tags:
    None

Preferred version:
    2017.1.23     https://pypi.io/packages/source/c/certifi/certifi-2017.1.23.tar.gz

Safe versions:
    2017.1.23     https://pypi.io/packages/source/c/certifi/certifi-2017.1.23.tar.gz
    2016.02.28    https://pypi.io/packages/source/c/certifi/certifi-2016.02.28.tar.gz
```

Reply:
Good catch; I guess I clicked through Spack suggestion to fetch newer `py-certifi` without checksum when testing the thing locally.